### ISSUE TITLE
Extend for Multidimensional Arrays

### ISSUE THREAD

**Issue by jfischoff** [judah/vector-fftw#2]
It would be nice to have the multidimensional aspects of FFTW exposed.

**Comment by peddie** [comment]
I should have a patch for the 2- and 3-d stuff within the week, if there is still interest.  Work so far is here: https://github.com/peddie/vector-fftw/commit/aceb0ae31627b9e90da01884ca9e2153580f712a

**Comment by bgamari** [comment]
Any progress on this, @peddie?

**Comment by peddie** [comment]
Yes . . . but it is not all working yet and I'm travelling at the moment.
